src/Controller/Api/UserController.ts

```diff
@@ -47,4 +50,21 @@ export class UserController {
             }
         }
     }
+
+    public async getAllUserNodes(req: Request, res: Response) {
+        try {
+            const token = req.headers.authorization;
```

Comment:
This is case sensitive, it's better to use `Authorization` or even support both!